server/server.go

```diff
@@ -327,7 +327,7 @@ func configureStorage(conf *Config) (storageProvider storage.Store, err error) {
 		}
 	}
 
-	if conf.Persist {
+	if conf.Persist || conf.Snapshot {
```

Comment:
do we need to update `conf.Persist` to `true` so if subsequent code relies on the flag it would work as expected?